docs/docs/data-structures/tree/nodes.mdx

```diff
@@ -10,13 +10,16 @@ See [node types](./node-types.mdx) for details on the types of nodes that can be
 
 Below are some utilities provided to make working with nodes easier.
 
-### Node Information
+### `Tree.key`
```

Comment:
I'm thinking it'd be better to just have links to the api docs so we're not duplicating any info. It'd be even better if there's a way to embed the docs in pages. @Josmithr do you know if this is a thing?

Reply:
Yeah, I'm not sure what the best middle ground is between reducing duplication and facilitating using discovery/usage, but embedding or linking to the API docs sounds good.
I don't think we should do that as part of this PR though.